docs/examples/TernaryExamples.rst

```diff
@@ -0,0 +1,84 @@
+
+Calculating Isothermal Phase Diagrams
+=====================================
+
+Often in thermodynamics, it is useful to use a two dimensional plot to
+express equilibria in a three component system with fixed potentials.
+The ``ternplot`` API provides a straightforward way to calculate
+equilibria and generate ternary phase diagrams. Alternatively, if you
+already have an equilibrium calculation, you can directly import and use
+``eqplot``.
+
+ternplot
+--------
+
+Here we well use the ternplot API to plot an isothermal section of the
+Mg-Si-Zn system from the thermodynamic database found at the `NIST
+CALPHAD assessments <http://hdl.handle.net/11115/101>`__ assessed by
+Zhang et al. `Calphad 35
+(2011) <https://doi.org/10.1016/j.calphad.2011.09.008>`__. Note that we
+have commented out the Cu6Y phase from the database because it utilized
+the Cu2 species. Species are not currently supported in pycalphad.
+
+.. code:: ipython3
+
+    %matplotlib inline
+    from pycalphad import Database, ternplot
+    from pycalphad import variables as v
+    
+    db_al_cu_y = Database('Al-Cu-Y.tdb')
+    comps = ['AL', 'CU', 'Y', 'VA']
+    phases = list(db_al_cu_y.phases.keys())
+    conds = {v.T: 830, v.P:101325, v.X('AL'): (0,1,0.015), v.X('Y'): (0,1,0.015)}
+    
+    ternplot(db_al_cu_y, comps, phases, conds, x=v.X('AL'), y=v.X('Y'))
+
+
+
+
+.. parsed-literal::
+
+    <matplotlib.axes._subplots.TriangularAxesSubplot at 0x10d907518>
+
+
+
+
+.. image:: TernaryExamples_files/TernaryExamples_2_1.png
+
+
+triangular projection
```

Comment:
This example was confusing the first time I saw it because I didn't understand the plot was supposed to be nonsense. This would be improved with a contour plot of a fictive custom property model.